packages/block-library/src/rich-image/rich-image/index.js

```diff
@@ -0,0 +1,357 @@
+/**
```

Comment:
Why is this a separate block?

Reply:
We originally worked on this in [Automattic/block-experiments](https://github.com/Automattic/block-experiments/tree/master/blocks/rich-image) where adding the functionality as a separate block was the only option.

#22581 is open for integrating it better and allowing it's usage for developers making other blocks.